webapp/channels/src/components/payment_form/payment_form.tsx

```diff
@@ -161,6 +164,20 @@ export default class PaymentForm extends React.PureComponent<Props, State> {
         if (changePaymentMethod) {
             paymentDetails = (
                 <React.Fragment>
+                    <div className='form-row'>
+                        <Input
+                            name='company_name'
+                            type='text'
+                            value={this.state.company_name}
+                            onChange={this.handleInputChange}
+                            onBlur={this.onBlur}
+                            placeholder={Utils.localizeMessage(
```

Comment:
Can you please try to use formatMessage with useIntl hoc for this component, we are trying to reduce dependency on these custom functions

Reply:
I opted to just convert the component to a functional one, been meaning to for a while anyways. Switched to formatMessage from react-intl.